### PR TITLE
OpenTelemetry ORM support information in FAQ.

### PR DESCRIPTION
**Pull request by balachandr**
OpenTelemetry support details.

### CODE REVIEW

**Comment by balachandr** [comment]
@odeke-em , please review and merge.

**Review by odeke-em — APPROVED** [review]
LGTM, thank you @balachandr! After this merge I’ll wait for a bit until my computer boots up then update the Google website too.

File: content/faq/_index.md [hunk 1]
```diff
@@ -77,3 +77,6 @@ Integration|action|controller|framework|route
 <a href="/python/psycopg2">psycopg2 <br />+ Flask</a>|<div style="text-align: center">&#10060;</div>|<div style="text-align: center">&#10004;</div>|<div style="text-align: center">&#10004;</div>|<div style="text-align: center">&#10004;</div>|<div style="text-align: center">&#10060;</div>
 <a href="/python/sqlalchemy">sqlalchemy<br />+ Flask</a>|<div style="text-align: center">&#10060;</div>|<div style="text-align: center">&#10004;</div>|<div style="text-align: center">&#10004;</div>|<div style="text-align: center">&#10004;</div>|<div style="text-align: center">&#10060;</div>
 <a href="/node/sequelize">sequelize.js<br />+ express.js</a>|<div style="text-align: center">&#10060;</div>|<div style="text-align: center">&#10004;</div>|<div style="text-align: center">&#10060;</div>|<div style="text-align: center">&#10060;</div>|<div style="text-align: center">&#10060;</div>
+
+### What ORMs supports OpenTelemetry ?
```
Comment (odeke-em) [inline content/faq/_index.md:81]: Minor nit: “ORMs support OpenTelemetry?”

File: content/faq/_index.md [hunk 2]
```diff
@@ -77,3 +77,6 @@ Integration|action|controller|framework|route
...
+Node.js ORMs(Knex, Sequelize, Express) supports OpenTelemetry.
```
Comment (odeke-em) [inline content/faq/_index.md:82]: Ditto here, please use “support” instead of “supports”

**Comment by balachandr** [comment]
Addressed your comment

**Pull request merged**